source/gmx/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+message(STATUS "Build GROMACS plugin")
+
+set(libgmxname ${LIB_DEEPMD_GROMACS})
+file(GLOB LIB_SRC src/*.cpp)
+file(GLOB INC_SRC include/*.h)
+
+add_library(${libgmxname} SHARED ${LIB_SRC})
+target_link_libraries(${libgmxname} ${LIB_DEEPMD_CC} ${LIB_DEEPMD} ${TensorFlow_LIBRARY} ${TensorFlowFramework_LIBRARY})
+target_include_directories(${libgmxname} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/../api_cc/include)
+target_include_directories(${libgmxname} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
+target_include_directories(${libgmxname} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../3rdparty/)
+
+set_target_properties(
+  ${libgmxname} 
+  PROPERTIES 
+  INSTALL_RPATH "$ORIGIN;${TensorFlow_LIBRARY_PATH}"
+)
+
+install (
+  FILES dp_gmx_patch
+  DESTINATION bin
+  PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
+)
+
+install (
+  DIRECTORY patches/
+  DESTINATION lib/patches
```

Comment:
NOT a good name. Suppose that a user install deepmd in `/usr/`, then the patches will be installed to `/usr/lib/patches`, which may conflict with other software. I would suggest changing it to something like `lib/deepmd_patches`

Reply:
`share` may be more proper than `lib`? See https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s11.html
> The /usr/share hierarchy is for all read-only architecture independent data files. 
> This hierarchy is intended to be shareable among all architecture platforms of a given OS; thus, for example, a site with i386, Alpha, and PPC platforms might maintain a single /usr/share directory that is centrally-mounted. Note, however, that /usr/share is generally not intended to be shared by different OSes or by different releases of the same OS.
>Any program or package which contains or requires data that doesn't need to be modified should store that data in /usr/share (or /usr/local/share, if installed locally). It is recommended that a subdirectory be used in /usr/share for this purpose. Applications using a single file may use /usr/share/misc.
>Game data stored in /usr/share/games must be purely static data. Any modifiable files, such as score files, game play logs, and so forth, should be placed in /var/games.

Reply:
Copy that